src/components/navbar/navbar-toggle.js

```diff
@@ -22,7 +22,7 @@ export const BNavbarToggle = /*#__PURE__*/ Vue.extend({
       default: () => getComponentConfig(NAME_NAVBAR_TOGGLE, 'label')
     },
     target: {
-      type: [String, Array],
+      type: [Array, String],
```

Comment:
@jackmu95 
What's the reasoning behind this change?

Reply:
Just named ordering.